abuild.in

```diff
@@ -837,6 +838,22 @@ check_maintainer() {
 	fi
 }
 
+check_license() {
+	local ret=0
+	local license_list=/usr/share/spdx/license.lst
```

Comment:
Do we already have a package shipping that file?

Reply:
https://pkgs.alpinelinux.org/packages?name=spdx-licenses-list&branch=edge